### PR TITLE
Tkt 00226 - Styling for page-specific ribbons and breadcrumbs

### PR DESCRIPTION
**Pull request by Android63**
These should be 30px wider on each side than the main content.

screen shot 2018-04-27 at 13 59 47

### CODE REVIEW

File: components/ribbons/_components.ribbons.scss [hunk 1]
```diff
@@ -23,6 +23,21 @@
...
+* rather than above the main content like other ribbons.
+* They need to jut out 30px either side of the main content.
+* 1. The top and bottom margins are adjusted so that the ribbon sits directy
+* under the header stripe.
+*/
+
+.c-ribbon--page-specific {
+  margin-top: -($global-spacing-unit + $global-border-mid); /* [1] */
+  margin-bottom: $global-spacing-unit + $global-border-mid; /* [1] */
+  margin-left: -$global-spacing-unit;
```
Comment (cehwitham) [inline components/ribbons/_components.ribbons.scss:37]: Is margin correct here? If we're trying to make them bigger, don't we need to add padding?

File: components/breadcrumb/_components.breadcrumb.scss [hunk 2]
```diff
@@ -7,6 +7,7 @@
  *
  * 1. Make sure all browsers scroll to the best of their abilities.
  * 2. Native-like scrolling on touch devices.
+ * 3. Use negative top margin to negate bottom margin of header this is under.
```
Comment (cehwitham) [inline components/breadcrumb/_components.breadcrumb.scss:10]: Does this work with or without a ribbon in place?

File: components/ribbons/_components.ribbons.scss (hunk 1 above)
Comment (Android63) [reply components/ribbons/_components.ribbons.scss:37]: No. The negative top margin is to stop the ribbon being pushed down by the bottom margin of the header. Hope that makes sense!

File: components/breadcrumb/_components.breadcrumb.scss (hunk 2 above)
Comment (Android63) [reply components/breadcrumb/_components.breadcrumb.scss:10]: Yes. I've tested all four permutations.

**Comment by Android63** [comment]
@cehwitham I haven't made any changes but have answered your questions. Could you review this again please?

**Review by cehwitham — APPROVED** [review]

**Pull request merged**